client/overview/task-list/dispute-tasks.ts

```diff
@@ -0,0 +1,221 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import moment from 'moment';
+import { getHistory } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import type { TaskItemProps } from './types';
+import type { CachedDispute } from 'wcpay/types/disputes';
+import { formatCurrency } from 'wcpay/utils/currency';
+import { getAdminUrl } from 'wcpay/utils';
+import wcpayTracks from 'wcpay/tracks';
+
+/**
+ * Returns true if the dispute is due within the specified number of days.
+ *
+ * @param {CachedDispute} dispute - The dispute to check.
+ * @param {number} [days] - The number of days to check. Defaults to 1.
+ *
+ * @return {boolean} True if the dispute is due within the specified number of days.
+ */
+const isDueWithin = ( dispute: CachedDispute, days = 1 ) => {
```

Comment:
`nit`: I don't see the reason we need the 2nd argument, `days`, to have default value here. All usages pass the number of days.

```suggestion
const isDueWithin = ( dispute: CachedDispute, days ) => {
```

Reply:
Good catch! Making this a required arg reduces complexity 👍

Updated in a7ef1e5f736ebd18234f9e23440746a83af264ed